packages/lesmis/src/views/VitalsView.js

```diff
@@ -53,9 +51,9 @@ const VitalsContainer = styled.div`
 /* eslint-disable react/prop-types */
 // Vitals data is essentially unstructured so no point checking prop types
 
-const CountryView = ({ vitals }) => (
+const CountryView = ({ vitals, variant }) => (
```

Comment:
Is this variant option used anywhere? To me it seems un used?

Reply:
It has been used while I was coding, not anymore. Have been removed.